pymemcache/client/base.py

```diff
@@ -109,6 +109,22 @@ def _check_key(key, allow_unicode_keys, key_prefix=b''):
     return key
 
 
+def _check_integer(value, name):
```

Comment:
What do you think about this function returning the value that we ultimately send to the server (instead of just checking its type)? That could save us a least a little work when this function detects we've been given a text input value.

Reply:
Yeah, I'll make that edit and mark this resolved when it's done.

Reply:
I ended up rebasing, since this conflicts mildly with #242. But I've changed `_check_integer` to do the encoding work.